marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "title": "Total number of open transactions",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 6411568736004342,
+            "layout": {
+                "height": 15,
+                "width": 47,
+                "x": 24,
+                "y": 26
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "[TODO] What query_read_rate and total",
+                "font_size": "14",
+                "show_tick": true,
+                "text_align": "left",
+                "tick_edge": "right",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 7562397926143496,
+            "layout": {
+                "height": 24,
+                "width": 18,
+                "x": 4,
+                "y": 53
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "avg:marklogic.servers.request_rate{$cluster}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Average rate of request across $cluster",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 7586296888730830,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 128,
+                "y": 69
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "If read query throughput is increasing while the hit rate for the compressed tree cache (`marklogic.forests.compressed_tree_cache_hit_rate`) is decreasing, it's likely that the cache is not large enough to handle the new queries\u2014consider adding memory to the cache.",
```

Comment:
I think we should remove the arrow and put this label above the graph it refers to. Right now, putting it to the right of the graph isn't very intuitive.